library/ssl_srv.c

```diff
@@ -3930,10 +3930,11 @@ static int ssl_parse_encrypted_pms( mbedtls_ssl_context *ssl,
      * 2 bytes of output, but we always read the first two bytes. It doesn't
      * matter in the end because diff will be nonzero in that case due to
      * peer_pmslen being less than 48, and we only care whether diff is 0.
```

Comment:
So actually, my comment wasn't quite right since I missed that `peer_pmslen` might also be uninitialized:
```suggestion
     * ret being nonzero, and we only care whether diff is 0.
```